src/xenia/gpu/graphics_system.cc

```diff
@@ -119,7 +119,7 @@ X_STATUS GraphicsSystem::Setup(cpu::Processor* processor,
   vsync_worker_running_ = true;
   vsync_worker_thread_ = kernel::object_ref<kernel::XHostThread>(
       new kernel::XHostThread(kernel_state_, 128 * 1024, 0, [this]() {
-        uint64_t vsync_duration = cvars::vsync ? 16 : 1;
+        uint64_t vsync_duration = cvars::vsync_interval; //cvars::vsync ? 16 : 1;
```

Comment:
I don't think the comment is intentional, is it?
This breaks the `vsync` cvar

Reply:
The comment is the old behavior.

Reply:
But the `vsync` cvar still exists and fulfills other purposes:
```
$ grep -nr cvars::vsync
src/xenia/gpu/command_processor.cc:400:  if (!cvars::vsync) {
src/xenia/gpu/command_processor.cc:916:        if (!cvars::vsync) {
src/xenia/gpu/graphics_system.cc:122:        uint64_t vsync_duration = cvars::vsync ? 16 : 1;
```
There should be a lower limit for the interval cvar because with vsync on and the interval pretty low (1), weird things can happen.

Reply:
Would it be ok to just disable vsync if the interval isn't 16?

Reply:
I would not change flags the user has set in the config
Rather do `vsync_duration = cvars::vsync ? std::max(5, cvars::vsync_interval) : 1;`

Reply:
This causes build errors: https://github.com/xenia-canary/xenia-canary/runs/2594659795#step:4:1244

Reply:
#include <algorithm>

Reply:
Does that go in .cc or .h?

Reply:
Std min is needed in cc only, so put it at top of cc only

Reply:
But it doesnt compile because it cannot deduce the type. What type is the cvar?

Reply:
uint64

Reply:
So do `uint64_t(5)` instead of just the number

Reply:
Or `std::max<uint64_t>(`